test/parallel/test-cli-bad-options.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common');
+
+// Tests that node exits consistently on bad option syntax.
+
+common.noop(); // Avoid lint errors if common is not required, or not used.
```

Comment:
Instead of doing this, you can just write `require('../common');` on the line 2 and don't assign the result of `require` anywhere.